### ISSUE TITLE
BUG: Binary columns are displayed incorrectly

### ISSUE THREAD

**Issue by aszenz** [beekeeper-studio/beekeeper-studio#1324]
I would expect to see the hex version of the binary data instead of this:
![Screenshot from 2022-08-01 17-04-00](https://user-images.githubusercontent.com/25319264/182179984-fd75a7be-b290-4259-96af-1afffab6e4bb.png)

**Comment by rathboma** [comment]
Yes it should be hex. Can you post some more info to help debug?:

- What database (engine and version)?
- What app version?
- Can you post an example `CREATE TABLE` and one or two `INSERT`s so we can replicate?

**Comment by aszenz** [comment]
* MYSQL 8.0.20
* 3.3.8
*
```sql
CREATE TABLE `id_bug` (
  `id` binary(16) NOT NULL, PRIMARY KEY (`id`)
) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_0900_ai_ci

insert into id_bug (id) values (uuid_to_bin("bf4ed8a6c48d11ecbacabaa3e4b5125e"))

select * from id_bug
```

**Comment by azmy60** [comment]
@aszenz  You can update to the newer version, it has been fixed.
